docs/getting-started-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,540 @@
+# Getting Started with Helm Charts (ScalarDB Analytics with PostgreSQL)
+
+This document explains how to get started with ScalarDB Analytics with PostgreSQL using Helm Chart on a Kubernetes cluster as a test environment. Here, we assume that you already have a Mac or Linux environment for testing. We use **minikube** in this document, but the steps we will show should work in any Kubernetes cluster.
```

Comment:
I think we can use some Linux distributions of WSL. In practice, I use Ubuntu on WSL2 on my laptop to test these steps.

Also, I think WSL is included in the `Linux environment`. So, I think we don't need to state WSL explicitly here. This is because if we mention a specific distribution, we have to test all distributions that are listed in this document.

This document explains very general steps on `minikube` or some kind of Kubernetes. So, I think we don't need to mention and test OS as a platform. It takes a lot of costs but there is no benefit to match it.